extensions/autoPepSIRF.py

```diff
@@ -211,6 +212,10 @@ def main():
 
         print(cmd)
         subprocess.run(cmd, shell=True)
+    # Turn off enriched scatterplots generation, as they cannot be run without a list of enriched peptides
+    elif args.enrichedScatterplots and not args.thresh and not args.pairs and not base:
```

Comment:
I don't think you need the bit after the first "and", and even if you wanted to include it, I don't think the logic is quite right.

What you really want to test is already being tested on line 206. So, this line can just be: 
"elif args.enrichedScatterplots:"

The elif covers the fact that the expression on line 206 evaluated to false. This bit just asks if args.enrichedScatterplots is set to True.